OHM-Trade-Agent-v1/tests/test_opip_streaming_windows_v1.py

```
@@ -0,0 +1,272 @@
+        asset="bitcoin", venue="BINANCE", timestamp_utc=NOW, window_seconds=1
+    )
+    window = empty_window(bounds).record_dropped_frame()
+    assert window.dropped_frame_count == 1
+    assert window.observation_count == 0
+
+
+# ------------------------------------------------------------------- routing
+
+
+def test_route_observation_new_window_when_none_held():
+    decision, bounds = route_observation(
```

Ruff (0.16.2)

[warning] Unpacked variable `bounds` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)